dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ReferenceConfig.java

```diff
@@ -418,7 +418,7 @@ private T createProxy(Map<String, String> map) {
             throw new IllegalStateException("Failed to check the status of the service " + interfaceName + ". No provider available for the service " + (group == null ? "" : group + "/") + interfaceName + (version == null ? "" : ":" + version) + " from the url " + invoker.getUrl() + " to the consumer " + NetUtils.getLocalHost() + " use dubbo version " + Version.getVersion());
         }
         if (logger.isInfoEnabled()) {
-            logger.info("Refer dubbo service " + interfaceClass.getName() + " from url " + invoker.getUrl());
+            logger.info(String.format("Refer %s service %s from url %s", StringUtils.isNotEmpty(getProtocol()) ? getProtocol() : "dubbo", interfaceClass.getName(), invoker.getUrl()));

```

Comment:
Here 'dubbo' doesn't mean protocol but 'dubbo service'. Since protocol is in url, I don't think it is necessary to emphasis it.

Reply:
OK, I just found it is easy to ignore the protocol of service from the log.